packages/SequenceTranslator/src/sdf-tab/sdf-tab.ts

```diff
@@ -56,7 +59,11 @@ export function saveSdf(
   ss: StrandData, as: StrandData, as2: StrandData, useChiral: boolean,
   oneEntity: boolean
 ): void {
-  if (ss.strand === '' && (as.strand !== '' && as2.strand !== '')) {
+  const nonEmptyStrands = [ss.strand, as.strand, as2.strand].filter((item) => item !== '');
+  if (
+    nonEmptyStrands.length === 0 ||
+    nonEmptyStrands.length === 1 && ss.strand === ''
+  ) {
```

Comment:
I guess it would be better to put the second part of logical operators in the brackets to make it more readable